shared/components/DiversityTickets/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
What's the rational of making a new module for this instead of putting it in the TicketPage? It's 11 lines.